### PR TITLE
fix failure in CliConfigTest if HELIOS_MASTER is set on host

### PR DESCRIPTION
**Pull request by mattnworb**
If you happen to run CliConfigTest#testConfigFromFile() on a host that
has the environment variable `HELIOS_MASTER` set, the test will fail as
it implicitly assumes no environment variables are set. To be safe when
running the test, always wipe out whatever env vars are inherited by the
process.

### CODE REVIEW

**Comment by mattnworb** [comment]
@davidxia @rohansingh @gimaker 

note the test failure seems to happen on master branch as well

**Comment by gimaker** [comment]
:+1:

Although I have to say that the general practice in that file seems very smelly. static variables don't get reset between tests, or even different test classes -- changing a static variable could potentially affect other tests that depend on it, and make it hard to figure out why things blow up.

**Comment by davidxia** [comment]
@gimaker Do you mean static variables like `ENDPOINT1` and `SITE1`?

**Comment by gimaker** [comment]
@davidxia Sorry I was referring to mutating static *non-final* variables, like the `CliConfigTest` does.

File: helios-tools/src/test/java/com/spotify/helios/cli/CliConfigTest.java [hunk 1]
```diff
@@ -33,6 +34,14 @@
 
   @Rule public final ExpectedException expectedEx = ExpectedException.none();
 
+  @Before
```
Comment (gimaker) [inline helios-tools/src/test/java/com/spotify/helios/cli/CliConfigTest.java]: If you want to make sure `environment` is always empty unless explicitly set, should you not do this in `@After` also?

**Comment by mattnworb** [comment]
I was considering rewriting the test for the same static var reasons (it also builds a bunch of Json with string literals), decided to do that in a separate PR.

File: helios-tools/src/test/java/com/spotify/helios/cli/CliConfigTest.java (hunk 1 above)
Comment (mattnworb) [inline helios-tools/src/test/java/com/spotify/helios/cli/CliConfigTest.java]: yes very true

**Pull request merged**